tomcat-10.1.yaml

```diff
@@ -35,78 +40,77 @@ pipeline:
       compile.debug=false
       EOF
 
-  - runs: |
-      ant
+subpackages:
+  - range: openjdk-versions
+    name: ${{package.name}}-${{range.value}}
+    dependencies:
+      runtime:
+        - ${{range.value}}-default-jvm
+        - tomcat-native
+    pipeline:
+      - runs: |
+          export JAVA_HOME="/usr/lib/jvm/java-17-openjdk"
```

Comment:
java-${{range.key}}

Reply:
Tomcat builds with OpenJDK 17, but supports 11-21 at runtime. We benefit from tests against all JVM versions supported at runtime, but we do end up building duplicate packages

Reply:
ah ok can you add a comment to that effect!

Reply:
Yes, I can definitely do that :slightly_smiling_face: